core/tests/test_utils.py

```diff
@@ -779,15 +902,20 @@ class FunctionWrapper(object):
     def __init__(self, func):
         """Creates a new FunctionWrapper instance.
 
-        args:
-          - func: a callable, or data descriptor. If it's a descriptor, its
+        Args:
+            func: a callable, or data descriptor. If it's a descriptor, its
             __get__ should return a bound method. For example, func can be a
             function, a method, a static or class method, but not a @property.
         """
         self._func = func
         self._instance = None
 
     def __call__(self, *args, **kwargs):
+        """Overrides the call method for the function to call pre_call_hook
+        method which would be called (for parity with your description of
```

Comment:
Why the "(for parity with your description of post_call_hook)" part?

Reply:
Done!